worker/src/main/java/gov/cms/ab2d/worker/bfdhealthcheck/BFDHealthCheck.java

```diff
@@ -73,6 +77,7 @@ private void updateMaintenanceStatus(Status status, String statusString) {
         PropertiesDTO propertiesDTO = new PropertiesDTO();
         propertiesDTO.setKey(MAINTENANCE_MODE);
         propertiesDTO.setValue(statusString);
+        slackLogger.logErrorMsg("Maintenance Mode status: " + statusString);
```

Comment:
How often is this run? Might want to make sure this doesn't become spam?

Reply:
Only when there is a change to the maintenance mode which doesn't happen unless it receives the same response more than x number of times. I guess we'll find out.